src/main/resources/application-test.yml

```
@@ -0,0 +1,29 @@
+  datasource:
+    url: jdbc:h2:mem:testdb
+    driverClassName: org.h2.Driver
+    username: sa
+    password: ""
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
+    database: h2
+    show-sql: false
+  main:
+    banner-mode: off
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)